### ISSUE TITLE
For chip with input box, the input is not usable when there're a lot of chips, and zoom to 500%

### ISSUE THREAD

**Issue by tinayuangao** [angular/material2#6077]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Chip list input box still usable with lots of chips & zoom to 500%

#### What is the current behavior?
User cannot input anything if there're a lot of chips and zoom to 500%

#### What are the steps to reproduce?
In input box of chip list, enter a lot of chips and zoom to 500%

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U
